spec/support/builders.rb

```diff
@@ -365,7 +365,6 @@ def build_with(builder, name, args, &blk)
       versions = args.last || "1.0"
       spec     = nil
 
-      # TODO: [2.0] Change this, since we won't use options[:path]
       options[:path] ||= @_build_path
```

Comment:
No. It looks like this options hash is specific to this file and has nothing to do with the options hash passed to `bundle` commands.